.msggen.json

```diff
@@ -4196,70 +4185,6 @@
             "added": "pre-v0.10.1",
             "deprecated": false
         },
-        "Fundchannel_Start": {
-            "added": "pre-v0.10.1",
-            "deprecated": null
-        },
-        "Fundchannel_Start.amount": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.announce": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.channel_type": {
-            "added": "v24.02",
-            "deprecated": false
-        },
-        "Fundchannel_Start.channel_type.bits[]": {
-            "added": "v24.02",
-            "deprecated": false
-        },
-        "Fundchannel_Start.channel_type.names[]": {
-            "added": "v24.02",
-            "deprecated": false
-        },
-        "Fundchannel_Start.channel_type[]": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.close_to": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.feerate": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.funding_address": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.id": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.mindepth": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.push_msat": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.reserve": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.scriptpubkey": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.warning_usage": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
```

Comment:
Oops, this looks like it'll need a rebase. Let me try to rebase it myself, and see if we end up with this as well.

Reply:
sorry about this one, we merged a bad .msggen.json into master where i generated Fundchannel and FundChannel. So every pr now has these deletions in them.

Reply:
but the deletions are merged aswell already so...

Reply:
Ok, got it. As long as the IDs don't change in-between releases, that's fine too. :+1: